src/extension.ts

```diff
@@ -110,6 +122,14 @@ async function activateYmlContributor(context: vscode.ExtensionContext) {
         }
     }));
 
+    // Re-request the schema when sessions change since auto-detection is dependent on
+    // being able to query ADO organizations using session credentials.
+    context.subscriptions.push(azureAccountApi.onSessionsChanged(async () => {
```

Comment:
This can be included in the onSessionsChanged hook above - just change the status to check for both LoggedIn & LoggedOut :)

Reply:
Done